src/chocolatey/infrastructure.app/services/ChocolateyPackageService.cs

```diff
@@ -628,7 +628,11 @@ public virtual void outdated_run(ChocolateyConfiguration config)
  Output is package name | current version | available version | pinned?
 ");
 
-            config.PackageNames = ApplicationParameters.AllPackages;
+            if (config.PackageNames.Length == 0)

```

Comment:
I thought at first we'd need to go back to handle additional argument parsing to ensure config.PackageNames is set, but I noticed the handle arg parsing method for the outdated command is already doing that. 👍